object_analytics_node/launch/rviz/default.rviz

```diff
@@ -122,7 +122,7 @@ Visualization Manager:
   Enabled: true
   Global Options:
     Background Color: 48; 48; 48
-    Fixed Frame: camera_depth_optical_frame
+    Fixed Frame: camera_color_optical_frame
```

Comment:
It would be better to keep the previous frame id since of point cloud is more closer to depth than color.

Reply:
after aligned depth , frame of it should be the same as color